captcha/.gitignore

```diff
@@ -0,0 +1 @@
+\.idea/
```

Comment:
Is this .gitignore necessary? Can't we ignore all of `.idea`?

Reply:
👍